src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -277,6 +278,14 @@ const formConfig = {
           uiSchema: unemployabilityFormIntro.uiSchema,
           schema: unemployabilityFormIntro.schema,
         },
+        medicalCareHistory: {
```

Comment:
Is there something else we can call `medicalCareHistory` to make it more related to 8940?

Reply:
changed!